# generated by patch-package 6.5.0 on 2021-12-13 14:17:10
#
# command:
#   npx patch-package react --exclude never
#
# declared package:
#   react: 18.0.0-rc.0
#
diff --git a/node_modules/react/package.json b/node_modules/react/package.json
index eb7032d..382d476 100644
--- a/node_modules/react/package.json
+++ b/node_modules/react/package.json
@@ -30,6 +30,7 @@
       "default": "./index.js"
     },
     "./build-info.json": "./build-info.json",
+    "./package.json": "./package.json",
     "./jsx-runtime": "./jsx-runtime.js",
     "./jsx-dev-runtime": "./jsx-dev-runtime.js",
     "./": "./"
